### DISCUSSION TITLE
Response's user object

### DISCUSSION THREAD

**Discussion by Brune04** [a-tokyo/react-apple-signin-auth#70] [Q&A] [answered]
Hi,

You say that "onSuccess response object will contain the user object on the first time attempt only. Meaning if you make another signIn attempt for the same account you will not get the user object." Is there no other way to see what is in this object? I did a successful login, but didn't see what was in the return the first time. Am I just out of luck now? Is there no way to change this or reset it so I can see the user object again?

**Comment by a-tokyo** [comment]
Hey @Brune04

There's a way to reset this by logging in to your apple ID -> go to settings -> go to `Password and Security` -> Apps Using Apple ID ->  Select your app -> Click on `Stop using Apple ID`.

**Comment by a-tokyo** [comment, marked as answer]
Hey @Brune04

There's a way to reset this by logging in to your apple ID -> go to settings -> go to Password and Security -> Apps Using Apple ID -> Select your app -> Click on Stop using Apple ID.
